inst/examples/e2e-test/app.R

```diff
@@ -4,14 +4,18 @@ library(shiny.fluent)
 ui <- fluidPage(
   fluentInputsUI("fluentInputs"),
   shinyInReactUI("shinyInReact"),
-  scenariosUI("scenarios")
+  scenariosUI("scenarios"),
+  actionButtonsUI("actionButtons"),
+  reactInShinyUI("reactInShiny")
 )
 
 server <- function(input, output, session) {
   
   fluentInputsServer("fluentInputs")
   shinyInReactServer("shinyInReact")
   scenariosServer("scenarios")
+  actionButtonsServer("actionButtons")
+  reactInShinyServer("reactInShiny")
   
```

Comment:
Please remove the empty lines at the beginning and end (I don't think they make the code more readable in this case).